src/Native/System.Private.CoreLib.Native/configure.cmake

```diff
@@ -60,6 +62,20 @@ int main(void)
 }" HAVE_SCHED_GETCPU)
 set(CMAKE_REQUIRED_LIBRARIES)
 
+if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
It can, but we'll have to make it what I put there as soon as someone starts bringing up BSD. I put the structure in place that CoreCLR has in `configure.cmake` (I just didn't actually write the BSD specific code because I don't like to write code I can't test). I can roll that back if you still want me to.